distribution/docs/src/main/resources/_contents/_geowebcache-contents/managing-geowebcache-contents.adoc

```diff
@@ -0,0 +1,127 @@
+${ddf-geowebcache} enables a server providing a tile cache and tile service aggregation.
+See (http://geowebcache.org[GeoWebCache]) for more information.
+This application also provides an administrative plugin for the management of GeoWebCached layers.
+GeoWebCache also provides a user interface for previewing, truncating, or seeding layers at \https://localhost:8993/geowebcache/.
+
+=== Prerequisites for ${ddf-geowebcache}
+
+none.
+
+=== Installing ${ddf-geowebcache}
+
+The ${ddf-geowebcache} application is *not* installed by default. To install:
+
+* Navigate to the ${admin-console}.
+* Select *Manage*.
+* Select the application *Start* (icon:play[]) button. The application will move to *Active Applications* on startup.
+* Select *Back* in ${admin-console}.
+
+=== Configuring ${ddf-geowebcache}
```

Comment:
The instructions are in the following subsections, but I added a sentence to clarify